cmsis/TARGET_CORTEX_A/core_ca.h

```diff
@@ -1372,18 +1372,18 @@ __STATIC_INLINE void GIC_DistInit(void)
   GIC_SetPriority((IRQn_Type)0, 0xff);
   priority_field = GIC_GetPriority((IRQn_Type)0);
 
-  for (i = (IRQn_Type)32; i < num_irq; i++)
+  for (i = 32; i < num_irq; i++)
   {
       //Disable the SPI interrupt
-      GIC_DisableIRQ(i);
+      GIC_DisableIRQ((IRQn_Type)i);
       if (i > 15) {
```

Comment:
i starts at 32, thus this condition is always true.
We fixed GIC implementation in latest CMSIS release.